test/test_configs/config.mysql.test.yml

```diff
@@ -6,3 +6,4 @@ mocks:
     debug: false
     dbName: myDbName
     version: "8"
+    expire_after: 1m30s
```

Comment:
is this really required or is it just a test?

Reply:
I'm just testing that it works.

Reply:
1m should be more than enough for this test